modules/nf-core/dragmap/hashtable/environment.yml

```diff
@@ -4,4 +4,5 @@ channels:
   - conda-forge
   - bioconda
 dependencies:
-  - bioconda::dragmap=1.3.0
+  # WARN: Do not update this tool to 1.3.0 until https://github.com/Illumina/DRAGMAP/issues/47 is resolved
+  - dragmap=1.2.1
```

Comment:
```suggestion
  - biocomda::dragmap=1.2.1
```

Reply:
I was being consistent by removing it there, but I should had that to the other file?